packages/google_maps_flutter/google_maps_flutter/pubspec.yaml

```diff
@@ -23,19 +23,19 @@ dependencies:
   google_maps_flutter_android: 
     git:
       url: https://github.com/fulpismo/plugins.git
-      ref: v2.2.3-fork.4
+      ref: feat/animated-markers
```

Comment:
minor issue for now, but what do you think about using the same naming conventions from the tags in this PR as well?
you could name it `v2.2.3-fork.5` and after merging it, we create a new release with the same name (and we don't have to change it in another commit or PR)
alternatively you can also change it to `main-fork` so all packages point to the target branch when using this version temporarily.

Reply:
Oh, I will change that! Thanks!